README.md

```diff
@@ -85,19 +85,19 @@ qiime tools import \
 
 This package is not actively being maintained. If you are interested in contributing, please reach out!
 
-- output membership info          
-    - first attempt will be just to print membership to stdout [done]     
-    - in the future, will want to define a new file format and write membership to that      
-    - in the more distant future, perhaps there could even be a way to visualize that membership       
+- output membership info
+    - first attempt will be just to print membership to stdout [done]
+    - in the future, will want to define a new file format and write membership to that
+    - in the more distant future, perhaps there could even be a way to visualize that membership
 
 ## Notes
 
-- The repo is called q2-dbotu to keep in line with other qiime 2 plugin repo names, but the function is called dbotu_q2 so that it (1) does not conflict with the existing pip package called `dbotu` and (2) shows up alphabetically under `dbotu` which is where I assume most humans will be looking for it in the qiime plugin listings.     
+- The repo is called q2-dbotu to keep in line with other qiime 2 plugin repo names, but the function is called q2_dbotu so that it (1) does not conflict with the existing pip package called `dbotu` and (2) shows up alphabetically under `dbotu` which is where I assume most humans will be looking for it in the qiime plugin listings.
```

Comment:
```suggestion
```

I think we can just delete this note, right? Since the function now shows up as `dbotu`?

Side note: how come the function shows up as dbotu and not q2_dbotu?? Is QIIME stripping the q2 prefix under the hood?